### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by imran5644**
Add the logic folder with methods:
1) Calculate method
2) Operate method
3) Import calculate on App component
4) Import operate on calculate

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @imran5644**

Great job with the project so far. calculate.js is imported in the app.js file, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] in the calculate function, make sure to include every case such as when you click on a number, when you click on the '=' button, when you click on any of the other operators.
- [x] in the operate function, make sure to include a case for the '%' button.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @imran5644, great job on this project :fireworks: The "calculate" function and the "operate" function have been addressed :+1: However, I have realized that in the "operate" function we are missing an important requirement that needs to be addressed before proceeding.

- Kindly take a closed look at this requirement
`operate` switches `operation` and uses `plus`, `minus`, `times`, `div` and `modifier` (turning the number into a percentage based number, `e.g. 95 % == .95`) functions. So instead of using our normal mathematical operation sign as currently, let us rather use the methods above

Once this is resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by imran5644** [comment]
@addod19 I am done with the changes. so please approve my project.

File: src/logic/operate.js [hunk 1]
```diff
@@ -9,27 +9,27 @@ const operate = (firstNum, secondNum, operation) => {
...
-    case '/':
-      total = bigFirstNum / bigSecondNum;
+    case '÷':
+      total = bigFirstNum.div(bigSecondNum);
       result = { total, next: total, operation };
       break;
 
     case '%':
-      return { total: bigFirstNum, next: bigSecondNum / 100, operation };
+      return { total: bigFirstNum, next: bigSecondNum.div(100), operation };
```
Comment (addod19) [inline src/logic/operate.js]: Kindly use `mod` here instead of the `div(100)`. This is all left to approve the project :smile:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @imran5644, Awesome job sor far!!! We are almost there. Kindly make a little adjustment to the `operate.js` file in line `32` as in the comment below:
Make use of the `mod()` method as mentioned in the previous review

Once this is done, we are good to go :rocket: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**

- [ ] OPTIONAL - This should be required but has been added as an optional change. You were right the first time I apologize for the mixup in the previous review. Please consider changing the `.mod` back to `.div(100)` because the `%` is a percentage operator, not a modifier operator.

Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**